pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -272,6 +273,89 @@ func (b *OFBridge) IsConnected() bool {
 	return b.ofSwitch.IsReady()
 }
 
+func (b *OFBridge) AddFlowsInBundle(addflows []Flow, modFlows []Flow, delFlows []Flow) error {
+	// If none Openflow entries are requested to be added or modified or deleted on OVS, return.
+	if len(addflows) == 0 && len(modFlows) == 0 && len(delFlows) == 0 {
+		klog.V(2).Infoln("No Openflow entries need to be synced to the OVS bridge, returning.")
+		return nil
+	}
+	// Create a new transaction.
+	tx := b.ofSwitch.NewTransaction(ofctrl.Atomic)
+	// Open a bundle on the OFSwitch.
+	if err := tx.Begin(); err != nil {
+		return err
+	}
+
+	syncFlows := func(flows []Flow, operation int) error {
+		for _, flow := range flows {
+			ofFlow := flow.(*ofFlow)
+			ofFlow.Flow.NextElem = ofFlow.lastAction
+			// "AddFlow" operation is async, the function returns error when constructing and sending BundleAdd message.
+			// It doesn't represent all Openflow entries are added into the bundle even if the functionreturns nil.
+			// The number of the Openflow entries successfully added into the bundle is returned from function "Complete".
+			flowMod, err := ofFlow.Flow.GenerateFlowModMessage(operation)
```

Comment:
I'm taking the liberty to rewrite the comment above since there are a few typos. May I suggest the following:
> "AddFlow" operation is async, the function only returns errors which occur when constructing and sending the BundleAdd message. An absence of error does not mean that all OpenFlow entries have been added to the bundle by the switch. The number of entries successfully added to the bundle by the switch will be returned by function "Complete".